keras/src/layers/preprocessing/image_preprocessing/resizing.py

```diff
@@ -112,10 +118,78 @@ def transform_segmentation_masks(
     def transform_labels(self, labels, transformation=None, training=True):
         return labels
 
+    def get_random_transformation(self, data, training=True, seed=None):
+
+        if isinstance(data, dict):
+            input_shape = self.backend.shape(data["images"])
+        else:
+            input_shape = self.backend.shape(data)
+
+        input_height, input_width = (
+            input_shape[self.height_axis],
+            input_shape[self.width_axis],
+        )
+        if input_height is None or input_width is None:
```

Comment:
This would never actually occur; in the TF dynamic shape case you'd get int tensors in the shape